src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -35,6 +35,11 @@ private IdentityModelEventSource()
         /// </summary>
         public static bool ShowPII { get; set; } = false;
 
+        /// <summary>
+        /// Flag which indicates whether or not armed security artifact is shown in logs when <see cref="ShowPII"/> is set to true. False by default.
```

Comment:
>true [](http://example.com/codeflow?start=126&length=4)

It is not clear for someone without context what a security artifact is. Let's provide a concrete example, security tokens

Reply:
@ciaozhang

Reply:
Updated the XML.